### PR TITLE
Fix/shape

### PR DESCRIPTION
**Pull request by Hendrixer**
## Clickup issues
[#rn0p0](https://app.clickup.com/t/rn0p0)
[#rn0mk](https://app.clickup.com/t/rn0mk)

#rn0mk[closed]
#rn0p0[closed]

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 export * from './shape'
 export * from './page'
-export * from './prepare'
-export * from './system-shapes'
+export * from './schema'
+export * from './utils/systemModels'
```
Comment (traversng) [inline src/index.ts]: 👓

File: src/types.ts [hunk 2]
```diff
@@ -62,29 +66,51 @@ export interface IPage {
...
   apiId: string
   name: string
   route: string
 }
 
-export type ShapeValidator = (
-  shape: IShape,
-  shapes: IShape[]
-) => IShapeValidation[]
+export interface IShapeOptions {
```
[the remark this answers is not in the document]
Comment (Hendrixer) [reply src/types.ts]: Page and Shape are types of models. This interface is for shapes

File: src/utils/__tests__/fieldTypes.spec.ts [hunk 3]
```diff
@@ -0,0 +1,98 @@
+import { fieldTypes, addMetaFields, createField } from '../fieldTypes'
+import { types } from '../constants'
```
[the remark this answers is not in the document]
Comment (Hendrixer) [reply src/utils/__tests__/fieldTypes.spec.ts]: fieldTypes is a chainable config like propTypes that creates the entire field object for you. types was just some props with strings

**Pull request merged**